### PR TITLE
Issue #75 - Room settings tags

### PR DESCRIPTION
**Pull request by oobjectt**
Tags has come to Nitro!
Added the 'YoVoy' field in the room settings.

imagen

If the user types an inappropriate word or restricted words, the user will send a message that the tag cannot be put on.

imagen

When you click on the tag information, it will automatically redirect to the browser and look for the 'YoVoy' filter

imagen

### CODE REVIEW

File: src/api/navigator/RoomSettingsErrorType.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export class RoomSettingsErrorType
```
Comment (dank074) [inline src/api/navigator/RoomSettingsErrorType.ts:1]: You don't need to create these enums, they are already defined in `RoomSettingsSaveErrorParser` in the renderer

File: src/components/navigator/views/room-settings/NavigatorRoomSettingsBasicTabView.tsx [hunk 2]
```diff
@@ -1,30 +1,52 @@
...
+    useMessageEvent<RoomSettingsSaveErrorEvent>(RoomSettingsSaveErrorEvent, event =>
+    {
+        const parser = event.getParser();
+
+        if (!parser) return;
+
+        switch (parser.code)
+        {
+            case RoomSettingsErrorType.ROOM_TAGS_BADWWORDS:
+            case RoomSettingsErrorType.RESTRICTED_TAGS:
```
Comment (dank074) [inline src/components/navigator/views/room-settings/NavigatorRoomSettingsBasicTabView.tsx:41]: use the values provided in `RoomSettingsSaveErrorParser` (for example `RoomSettingsSaveErrorParser.ERROR_INVALID_TAG`. Also make sure you use the correct localization for each error type (you can check the localization for each error type here: https://github.com/dank074/HabboSwfOpenSource/blob/5bad64a6704e38576c96c08a7a3a246305e31a92/src/com/sulake/habbo/navigator/roomsettings/RoomSettingsCtrl.as#L336)

**Review by dank074 — CHANGES_REQUESTED** [review]

**Comment by oobjectt** [comment]
Made !

Thank you very much for your advice, also thanks for the page, I was unaware of it.

Thanks for all the feedback! 👍

**Review by dank074 — APPROVED** [review]
looks good

**Pull request merged**